runtime/JavaScript/src/antlr4/atn/ParserATNSimulator.js

```diff
@@ -255,6 +255,7 @@ var PredictionContext = require('./../PredictionContext').PredictionContext;
 var Interval = require('./../IntervalSet').Interval;
 var Transitions = require('./Transition');
 var Transition = Transitions.Transition;
+var AtomTransition = Transitions.AtomTransition;
```

Comment:
suggest to drop dumpDeadEndConfigs instead (comment says it's dead code)

Reply:
Done.